Automattic/vip
# feat(dev-env): Add `vip dev-env shell` command

## Description

This PR adds `vip dev-env shell` command.

Supported options:
  * `--slug name`: environment to operate on
  * `--root`: spawn a root shell (by default, log in as the `www-data` user)

It is also possible to run a custom command in a shell, like this:

```sh
vip dev-env shell -- whoami
```

## Steps to Test

* Create an start an environment, run `vip dev-env shell`, ensure you are in the container.
* Run `vip dev-env shell -- whoami`, ensure the output is `www-data`.
* Run `vip dev-env shell --root -- whoami`, ensure the output is `root`.

E2E tests check last two scenarios; the first one has to be tested manually.
